app/sass/style.css

```diff
@@ -0,0 +1,14 @@
+body #field {
+  width: 100%;
+  height: 865px;
```

Comment:
It'f very big value
![image](https://user-images.githubusercontent.com/58324308/75687208-7da12880-5cae-11ea-98ba-3cc54af77334.png)